packages/kit/types/hooks.d.ts

```diff
@@ -20,11 +20,21 @@ export interface GetSession<Locals = Record<string, any>, Session = any> {
 	(request: ServerRequest<Locals>): MaybePromise<Session>;
 }
 
+export interface HandleInput<Locals = Record<string, any>> {
+	request: ServerRequest<Locals>;
+	resolve: (request: ServerRequest<Locals>) => MaybePromise<ServerResponse>;
+}
+
```

Comment:
we don't need separate `SequenceHandle` types if we get rid of `next` — let's just revert all this

```suggestion
```

Reply:
Deleted the commit which added the `next` function.